core/app/models/spree/payment_method/check.rb

```diff
@@ -15,17 +15,27 @@ def can_void?(payment)
     end
 
     def capture(*args)
-      ActiveMerchant::Billing::Response.new(true, "", {}, {})
+      simulated_successful_billing_response
     end
 
     def cancel(response); end
 
     def void(*args)
-      ActiveMerchant::Billing::Response.new(true, "", {}, {})
+      simulated_successful_billing_response
     end
 
     def source_required?
       false
     end
+
+    def credit(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `credit(*)` if you want the method to accept any arguments but don't care about them.